go.sum

```diff
@@ -1,4 +1,3 @@
-bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898 h1:SC+c6A1qTFstO9qmB86mPV2IpYme/2ZoEQ0hrP+wo+Q=
```

Comment:
why does the port change PR introduce that many go.sum changes?

Reply:
I did `go clean -modcache` before `go mod tidy`, and I did it twice to make sure `go.sum` needs to be updated. So I was guessing master branch might be not clean.

Reply:
feel free to update the makefile if our `make reviewable` not accurate